src/css/styles/_general.scss

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-display: swap;
+  font-family: "Praktika Bold";
+  src: url("fonts/praktika-boldextended-webfont.eot");
+  src:
+    url("fonts/praktika-boldextended-webfont.eot?#iefix") format("embedded-opentype"),
+    url("fonts/praktika-boldextended-webfont.woff2") format("woff2"),
+    url("fonts/praktika-boldextended-webfont.woff") format("woff"),
+    url("fonts/praktika-boldextended-webfont.ttf") format("truetype"),
+    url("fonts/praktika-boldextended-webfont.svg#praktikabold_extended") format("svg");
+  font-style: normal;
+  font-weight: normal;
+}
+
+@font-face {
+  font-display: swap;
+  font-family: "Praktika Medium";
+  src: url("fonts/praktika-medium-webfont.eot");
+  src:
+    url("fonts/praktika-medium-webfont.eot?#iefix") format("embedded-opentype"),
+    url("fonts/praktika-medium-webfont.woff2") format("woff2"),
+    url("fonts/praktika-medium-webfont.woff") format("woff"),
+    url("fonts/praktika-medium-webfont.ttf") format("truetype"),
+    url("fonts/praktika-medium-webfont.svg#praktikamedium") format("svg");
+  font-style: normal;
+  font-weight: normal;
+}
+
+.nav-mobile-kontakt {
+  font-family: "Praktika Medium", Arial, sans-serif;
+}
+
+.app {
+  font-family: "Praktika Bold", "Source Sans Pro", "Lucida Grande", sans-serif;
+  background-color: #fff;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+// ul li {
+//   list-style: none;
+// }
+
+.container-fluid {
+  max-width: 1600px;
+  padding: 0;
+  overflow: hidden;
+}
```

Comment:
Could we use `container-fluid` from bootstrap?

Reply:
This is container-fluid from bootstrap, just modified, so it matches out criteria

Reply:
OK, I understand it. But maybe we should use `container` instead of `container-fluid`, because the fluid one should be fullscreen and the regular one should have margin - and as far as I know we want margins, but the `max-width` of container is different.